.github/workflows/docs.yml

```diff
@@ -0,0 +1,15 @@
+name: docs
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - run: python -m pip install -r requirements.txt -r requirements-dev.txt
+      - run: mkdocs gh-deploy --force
```

Comment:
This is taken from https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions.
Not sure if it will work. Need to enable GitHub Pages from the repo settings for sure.

If we want to host under https://couler-proj.github.io instead of https://couler-proj.github.io/couler, we also need to create a `couler-proj.github.io` repo.